src/backend/templates/tracks.hbs

```diff
@@ -215,23 +215,23 @@
                               </div>
                             </div>
                             <div id="desc-{{session_id}}" class="collapse">
-                              {{{description}}}
-                              {{#if ../../../gcalendar.enabled}}
-                                <button class="btn btn-default add-to-calendar-button" onclick="main.handleAuthClick('{{title}}','{{location}}','{{calendarStart}}','{{calendarEnd}}','{{../../../eventurls.timezone}}','{{description}}')">Add to Calendar</button>
-                              {{/if}}
-                              {{#ifcontains audio "mp3"}}
-                                <audio controls="" preload="none" id="audio-block">
-                                  <source src="{{ audio }}">
-                                </audio>
-                              {{/ifcontains}}
-                              {{#ifcontains audio "aac"}}
-                                <audio controls="" preload="none" id="audio-block">
-                                  <source src="{{ audio }}">
-                                </audio>
-                              {{/ifcontains}}
-                              <hr class="clear-both">
                               <div id="speaker-{{session_id}}" class="session-speakers-list" aria-expanded="false"
```

Comment:
why are those lines moved around? Please only change lines that are related to the issue.

Reply:
@mariobehling These lines are moved to resolve issue #2253. I was waiting for this PR(for issue #2366) to get reviewed and in the meantime, I resolved issue #2253 and made a commit to the development branch, and pushed it to the origin. So the second commit change automatically got added in this PR. Shall I roll back the above change for now and raise another PR after this one gets merged?

Reply:
correct, only make changes in a PR which corresponds to the issue.